README.md

```diff
@@ -132,4 +133,4 @@ Pull requests are welcome. For major changes, please open an issue first to disc
 Please make sure to update tests as appropriate.
 
 ## License
-[MIT](https://choosealicense.com/licenses/mit/)
\ No newline at end of file
+[MIT](https://choosealicense.com/licenses/mit/)
```

Comment:
the text is the same!

I guess you have windows so new line char is different. or some other invisible char